rearrange virt section in client cfg (#437)
uyuni-project/uyuni-docs: modules/client-configuration/pages/vhm-nutanix.adoc

The Virtual Host Manager (VHM) runs on the {productname} Server.

Ensure you have installed the [systemitem]``virtual-host-gatherer-Nutanix`` package on the {productname} Server.

> To run a VHM, your {productname} Server will need to have port 9440 open, in order to access the Nutanix Prism Element API.

Reviewer: ```suggestion
To run a VHM, your {productname} Server must have port 9440 open to access the Nutanix Prism Element API.
```
